Formula/lib/libcouchbase.rb

```diff
@@ -1,11 +1,18 @@
 class Libcouchbase < Formula
   desc "C library for Couchbase"
   homepage "https://docs.couchbase.com/c-sdk/current/hello-world/start-using-sdk.html"
-  url "https://packages.couchbase.com/clients/c/libcouchbase-3.3.10.tar.gz"
-  sha256 "2c92b8dee1d8d02bab756023ff396cf93bf822400ee68f3a8128912a08348063"
+  url "https://github.com/couchbase/libcouchbase.git",
+      tag:      "3.3.10",
+      revision: "42c6854e68948e95e2bd30360e0eb287a31f592f"
   license "Apache-2.0"
   head "https://github.com/couchbase/libcouchbase.git", branch: "master"
 
+  livecheck do
```

Comment:
TODO: I believe the standard is to put in a comment in `livecheck` here explaining why to use GH releases instead of Git tags.